Assets/_IPS/Scenes/IPS-Thin.unity

```diff
@@ -121,6 +121,1421 @@ NavMeshSettings:
     debug:
```

Comment:
@RemyaTP 
The structure of the folders in Unity is incorrect.  The _MUTUAL folder exists to house assets which could conceivably live between CAS and IPS.  For now, move everything from it back into the _IPS folder.

Reply:
@KICO-SOURCE Moved